### ISSUE TITLE
Missing Bad Assignment explanation

### ISSUE THREAD

**Issue by piuccio** [jamesallardice/jslint-error-explanations#7]
Hello,
I love this site but I noticed that it lacks an explanation for `Bad assignment` and how to disable it.

The error is raised in cases like this

``` js
callAFunction() = 12;
```

Thanks
